x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ilm/WaitForRolloverReadyStep.java

```diff
@@ -236,19 +232,17 @@ public void evaluateCondition(Metadata metadata, Index index, Listener listener,
     RolloverRequest createRolloverRequest(String rolloverTarget, TimeValue masterTimeout, boolean rolloverOnlyIfHasDocuments) {
         RolloverRequest rolloverRequest = new RolloverRequest(rolloverTarget, null).masterNodeTimeout(masterTimeout);
         rolloverRequest.dryRun(true);
+        RolloverConditions.Builder builder = RolloverConditions.newBuilder(conditions);
```

Comment:
Now that we allow a condition to be set twice in the `RolloverConditions` we can use the builder here more efficiently.